package.json

```diff
@@ -44,6 +44,7 @@
     "less-loader": "2.2.3",
     "postcss-loader": "0.9.1",
     "raw-loader": "0.5.1",
+    "react-helmet": "3.1.0",
```

Comment:
This should be listed as a `peerDependency` for consuming apps to have installed. Also anything required for the demo app (this is) should be a `devDependency` not a `dependency`. Only thing in `dependencies` should be required webpack plugins for the config that gets decorated.